homeassistant/components/arlo/sensor.py

```diff
@@ -51,43 +82,44 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     sensors = []
     for sensor_type in config[CONF_MONITORED_CONDITIONS]:
```

Comment:
`SENSOR_TYPES` should be changed to a tuple and you can do:
```
for sensor_entry in SENSOR_TYPES:
  if sensor_entry.key not in config[CONF_MONITORED_CONDITIONS]:
    continue
...
```

Reply:
Done, but it did make test_sensor.py a bit more complex (due to basically faulty way of testing). However I prefer not to rewrite the test module in this PR.